### PR TITLE
Update using-azure-active-directory.md

### PR DESCRIPTION
**Pull request by MaxMeng1985**
When using MSI for authentication in ODBC driver:  For user assigned MSI, the UID should be set to the object id of the user assigned MSI.  For system assigned MSI, we don't have to set the UID, but if we set any value for UID, it also need to be the object id of the system assigned MSI.  The current document didn't make that clear enough, also the screenshot is a bit mis-leading by include a username in the UID field.

### CODE REVIEW

**Comment by Court72** [comment]
@David-Engel

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/connect/odbc/using-azure-active-directory.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ It's possible to use Microsoft Entra authentication options when creating or edi
 
 `Authentication=ActiveDirectoryMsi` for Microsoft Entra managed identity authentication
 
-![The DSN creation and editing screen with Managed Service Identity authentication selected.](windows/create-dsn-ad-msi.png)
+![The DSN creation and editing screen with Managed Service Identity authentication selected.](windows/create-dsn-ad-sys-msi.png)
```
Comment (David-Engel) [inline docs/connect/odbc/using-azure-active-directory.md:82]: I would revert this change and simply update the existing screenshot with one like this:

image

**Comment by rwestMSFT** [comment]
I'll do that, @David-Engel. Thanks for your contribution, @MaxMeng1985.
